### ISSUE TITLE
DAWN (no init.)

### ISSUE THREAD

**Issue by ZPP2000** [mxbastidasr/DAWN_WACV2020#8]
Im this paper, I notice DAWN (no init.), DAWN (no init.) mean what?

**Comment by beltegeuse** [comment]
It was an early experiment of pre-initializing the network with some specific weights to produce regular haar wavelet coefficients. This strategy could have helped to improve the network's accuracy. 

The default DAWN arguments are used inside the paper (+/- some hyperparameters explained in the results section).

**Comment by ZPP2000** [comment]
but  there is no Haar wavelet involved in the dawn network, only the lifting scheme to obtain high and low frequency information, DAWN (no init.) and DAWN (16 init.), 16 is the args.first_conv=16？

**Comment by ZPP2000** [comment]
Thank you for your answer:
I am from China. Some statements may not be accurate. Please forgive me 
I also have the following questions:
First, The training set is one of the four samples in each category as the training set (a total of 1188 pictures), and the rest as the test set (a total of 3564 pictures). The training accuracy is 100%. Why, is my data set wrong?
Second, in the visualization process, does the visualization part need major changes? I just adjust the parameter monkey to True
Third, the number of multi-scale layers is not set to one layer, why should it be set according to the size of the picture? Is one layer of information extraction inadequate?
Fourth, can this code implement classification on the MNist dataset and achieve good classification accuracy
Fifth, is this extracted feature strictly high and low frequency information

I look forward to your reply

**Comment by ZPP2000** [comment]
Thank you for your answer:
I am from China. Some statements may not be accurate. Please forgive me
I also have the following questions:
First, The training set is one of the four samples in each category as the training set (a total of 1188 pictures), and the rest as the test set (a total of 3564 pictures). The training accuracy is 100%. Why, is my data set wrong?
Second, in the visualization process, does the visualization part need major changes? I just adjust the parameter monkey to True
Third, the number of multi-scale layers is not set to one layer, why should it be set according to the size of the picture? Is one layer of information extraction inadequate?
Fourth, can this code implement classification on the MNist dataset and achieve good classification accuracy
Fifth, is this extracted feature strictly high and low frequency information

I look forward to your reply





曾盼盼&nbsp;
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mxbastidasr/DAWN_WACV2020"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月7日(星期五) 晚上10:16
***@***.***&gt;;
抄送:&nbsp;"曾盼盼 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [mxbastidasr/DAWN_WACV2020] DAWN (no init.) (Issue #8)





 
It was an early experiment of pre-initializing the network with some specific weights to produce regular haar wavelet coefficients. This strategy could have helped to improve the network's accuracy.
 
The default DAWN arguments are used inside the paper (+/- some hyperparameters explained in the results section).
